Podfile

```diff
@@ -6,9 +6,5 @@ platform :ios, '12.0'
 target 'MnemonicKitExampleApp' do
 end
 
```

Comment:
My idea here was that there were three targets:
- An example app, good for local develoment / messing about
- A static library, for consumption of dependent projects
- A test target, contains tests for static library. 

I see that you're moving the static library target to a framework? What is the rationale there? Where does the dependency for CryptoSwift get declared now?

Reply:
I think it's a standard to ship libraries like this as frameworks, at least from what I understand (I'm interested in your rationale why you chose static library).

Three targets are still there, I just deleted it from the pod, since, well, there are no pods 😄 
I should probably return it there, so it's not forgotten about. 

CryptoSwift is installed via Carthage and declared as a dependency. 🙂

Reply:
Ok, I did some research. Thanks for being patient, I'm not overly familiar with the tooling. 

Both Carthage and Cocoapods ultimately build shared libraries into frameworks. Also, on iOS it seems that frameworks are a bit more flexible in that they can bundle additional resources with code and they are easier to include in project files. 

I agree that it is then best to develop locally as a framework since that is what it is being shipped as. 

It is also worth noting that this is a change in the Podfile which declares the settings for the local xcode project. The .podspec file declares the settings to build a shared library. The CryptoSwift dependency still exists in the podspec, so I suspect that the pod will build and ship fine. 

However, I do not think that opening the project locally and building with Cocoapods will work as the Podfile no longer declares a CryptoSwift dependency. I think you may need to run `pod install` again to pick up changes after you've modified this file. Can you verify that you can still build the tests with CocoaPods?

Finally (and maybe this is obvious at this point) but Carthage just has the one cartfile which looks fine from here.

Reply:
No, running `pod install` will not work locally - but is that something that we would want to have? I do not see any benefits to it and looking at popular projects in the Swift community, most of them do not declare this kind of dependency twice. (declaring it in `podspec` should be more than fine).

Reply:
I didn't realize that but after poking around a bit more I agree with your sentiment. 

My understanding then is that to do development locally you would run:

```shell
$ carthage bootstrap --platform iOS --cache-builds
$ open open MnemonicKit.xcodeproj/
``` 

Correct?

Reply:
Yes, that is correct. Probably should add it to the manual.